english/shotgun/shotgun.md

```diff
@@ -0,0 +1 @@
+You cock the shotgun and blast the polar bear in the face. Brain matter flys all over the marshmallow wall. Suddenly the brain matter starts to disintegrate the wall. You see sunlight peeking through. After a while the wall is gone and you are free. 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.